Jenkinsfile

```diff
@@ -129,6 +140,7 @@ pipeline {
                 branch 'master'
```

Comment:
Question: what's the purpose of an `allOf ( anyOf (a, b, c, d) ) `, if there is only one block for the `allOf`? Would it be worth it to simplify it to simply use the `anyOf(a, b, c, d)`? Maybe I'm wrong 🤔

Reply:
I know that it's not related to the changes in the PR, but I've just seen same pattern in a few reviews today

Reply:
`allOf ( anyOf (a, b, c, d) )` is nto the case but `allOf ( anyOf (a, b, c, d) e)`:

There are two conditions in the `allOf`
- params.bench_ci is always required to be true and
- anyOf ( branch == master , tag ~= `v[0-9].*` , params.Run_As_Master_Branch, env.BENCHMARK_UPDATED != "false")

In other words, the bench stage is launched when an input parameter and either a branch/tag or changes in certain files of the source code.

Reply:
Oh oh I see, I did not see the second element in the AND. Thanks for clarifying!